hydra/_internal/core_plugins/importlib_resources_config_source.py

```diff
@@ -30,7 +30,7 @@ def load_config(
         if not res.exists():
             raise ConfigLoadError(f"Config not found : {normalized_config_path}")
 
-        with res.open() as f:
+        with open(res, encoding="utf-8") as f:
```

Comment:
did you see an actual problem here?
I think utf-8 is the default for importlib.